### DISCUSSION TITLE
倒数日、纪念日可以根据时间排序吗

### DISCUSSION THREAD

**Discussion by lengsi666** [WatsonYao/nian_app#109] [Q&A] [answered]
莴笋大大，nian的倒数日、纪念日可以根据时间排序吗
另外nian的标签可以一个记本一套标签吗

**Comment by WatsonYao** [comment]
倒数日是支持手动排序的，就没有时间排序了。
nian 的记本标签是共享的，写进展时添加标签时，会优先显示该记本下常用的标签。

**Reply by lengsi666** [reply]
那倒数日以后会考虑添加农历吗，生日提醒看的基本都是农历。。。。

**Reply by WatsonYao** [reply, marked as answer]
倒数日没有农历支持的，农历经常需要调整修补，是无法通过一次版本确定的。
农历生日的话，只能每年都手动添加了。

**Reply by lengsi666** [reply]
求求，调整修补也愿意等的，像原来日历一年一更新就可以的，求求

实在不行在日历上显示一下农历也可以
